package.json

```diff
@@ -59,6 +59,7 @@
     "babel-jest": "22.4.3",
     "babel-loader": "7.1.4",
     "babel-plugin-flow-runtime": "0.17.0",
+    "babel-plugin-module-resolver": "3.1.1",
```

Comment:
I think this shouldn't be in this PR, it's not in the scope of the issue IMHO

Reply:
oh my bad..fixing it right now..